config/routes.rb

```diff
@@ -1,3 +1,13 @@
+class RootConstraint
+  def initialize
+    @current = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
```

Comment:
Thinking about if it makes sense to just pass the conference as parameter

```
def initialize(conference)
  @current = conference
end
```
and then instantiate like this

```
conference = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
get '/' => redirect("/conferences/#{conference.first.short_title}"), constraints: RootConstraint.new(conference)
```